ci-operator/step-registry/optional-operators/configure-proxy-registry/optional-operators-configure-proxy-registry-commands.sh

```diff
@@ -40,6 +40,12 @@ echo "Configuring proxy registry : \"$OO_CONFIGURE_PROXY_REGISTRY\""
 # step-3: Disable the default OperatorSources/Sources (for redhat-operators, certified-operators, and community-operators) on your 4.5 cluster (or default CatalogSources in 4.6+) with the following command:
 oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
 
+# If ENABLE_REDHAT_OPERATOR_CATALOG is true, enable redhat-operators catalog
+if [[ "${ENABLE_REDHAT_OPERATOR_CATALOG}" == "true" ]]; then
+    oc patch OperatorHub cluster --type merge --patch '{"spec":{"sources":[{"disabled": false,"name": "redhat-operators"}]}}'
+    echo "redhat-operators catalog is enabled"
```

Comment:
@asergienk 
You would need to verify if the redhat-operators catalog is enabled or not. 
based on it you need to display the echo statement. 
Further, you need to consider the exit status of previous command too. 
pseudo code: 
```
if ( $? == 0 && oc get catsrc -n openshift-marketplace | grep redhat-operators | wc -l  == 1) {
    echo "redhat-operators catalog is enabled"
}
else {
    echo "redhat catalog not enabled please try again"
    exit 1
}
```